src/test/java/org/eolang/speco/MainTest.java

```diff
@@ -117,11 +127,14 @@ private List<String> exec(final String target) throws IOException {
             executor = "bash";
             flag = "-c";
         }
-        final Process process = new ProcessBuilder(executor, flag, command).start();
+        final Process process = new ProcessBuilder(
+            executor,
+            flag,
+            String.format("eoc link -s %s && eoc --alone dataize app && eoc clean", target)
+        ).start();
         final StringWriter writer = new StringWriter();
         IOUtils.copy(process.getInputStream(), writer);
         final String[] output = writer.toString().split("\\r?\\n");
-        final String[] result = Arrays.copyOfRange(output, 11, output.length - 1);
-        return Arrays.asList(result);
+        return Arrays.asList(Arrays.copyOfRange(output, 11, output.length - 1));
```

Comment:
@advasileva Why exactly `11`?

Reply:
@volodya-lombrozo `11` is a magic number that I counted myself. That's how many lines the `eoc` output takes up until the program output itself. As far as I know, this is the output only for `eoc link` and I can read the program output only for `eoc --alone dataize app`.

Reply:
@advasileva  It's better to add such description of `magic` numbers into JavaDoc.